src/views/components/attendees-list/attendees/attendee/ticket.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Generic: Success Order Page Attendee list - Attendee ticket template
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/components/attendees-list/attendees/attendee/ticket.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this                  [Global] Template object.
+ * @var Module           $provider              [Global] The tickets provider instance.
+ * @var string           $provider_id           [Global] The tickets provider class name.
+ * @var \WP_Post         $order                 [Global] The order object.
+ * @var int              $order_id              [Global] The order ID.
+ * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
+ * @var array            $attendees             [Global] List of attendees for the given order.
+ * @var array            $attendee              The current attendee.
+ */
+
+if ( empty( $attendee ) ) {
+	return;
+}
+
+$provider     = Tribe__Tickets__Tickets::get_ticket_provider_instance( $attendee['provider'] );
```

Comment:
If we already have the `$provider` as a variable defined above, that means it's defined already from the parent template. Why not use that?

Reply:
I found that with TC, that would work, but when using ETP with WC or EDD, the `provider` is a string. We'll leave as is for now since this was the original code. I just moved it into a new template.